collector/wifi_linux.go

```diff
@@ -74,6 +76,13 @@ func NewWifiCollector() (Collector, error) {
 			nil,
 		),
 
+		StationInfo: prometheus.NewDesc(
```

Comment:
Nope, I think I just did it out of habit. I'd be happy to unexport them all in a separate PR though.